### PR TITLE
Fixed C2D_RELEASE variable

### PR DESCRIPTION
**Pull request by ovk6**
Fixed C2D_RELEASE variable
Fixed "from"

### CODE REVIEW

File: versions.yaml [hunk 1]
```diff
@@ -26,13 +26,14 @@ versions:
...
       exporter: 'true'
+      c2dRelease: '8.0.0'
     tags:
       - '8.0-debian9'
       - '8-debian9'
       - '8.0'
       - '8'
       - 'latest'
-    from: *from
+    from: gcr.io/google-appengine/debian9
```
Comment (wgrzelak) [inline versions.yaml]: Revert to `*from` or remove `from` from `_variables`.

Comment (ovk6) [reply versions.yaml]: Newer versions have different "from". Removed from variables and added explicit from to older versions.

File: 7/debian9/7.9/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,5 @@
 FROM gcr.io/google-appengine/debian9 as downloader
+
 ARG SONAR_GPG=F1182E81C792928921DBCAB4CFCA4A29D26468DE
 ARG SONAR_VERSION=8.0
```
Comment (usmonster) [inline 7/debian9/7.9/Dockerfile]: Wrong version? (Since #16, I guess.)

Comment (ovk6) [reply 7/debian9/7.9/Dockerfile]: Fixed in #19

**Pull request merged**